cmd/limactl/disk.go

```diff
@@ -418,3 +421,62 @@ func diskResizeAction(cmd *cobra.Command, args []string) error {
 func diskBashComplete(cmd *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
 	return bashCompleteDiskNames(cmd)
 }
+
+// func newDiskRegisterCommand()
+func newDiskAddCommand() *cobra.Command {
+	diskAddCommand := &cobra.Command{
+		Use: "add existing DISK to Lima",
```

Comment:
Follow the syntax described in https://pkg.go.dev/github.com/spf13/cobra#Command